classes/Console/Command/ClearCacheCommand.php

```diff
@@ -39,19 +50,50 @@ public function execute(InputInterface $input, OutputInterface $output)
         $io->section('Clearing caches');
 
         $paths = [
-            $this->app['path']->cachePurifierPath(),
-            $this->app['path']->cacheTwigPath(),
+            $this->path->cachePurifierPath(),
+            $this->path->cacheTwigPath(),
         ];
 
         \array_walk($paths, function ($path) use ($io) {
-            \passthru(\sprintf('rm -rf %s/*', $path));
```

Comment:
I fiddled around too long trying to get this to work with `mikey179/vfsStream`, but without success . . .